src/main/java/io/appform/dropwizard/actors/actor/metadata/MessageMetadataGenerator.java

```diff
@@ -0,0 +1,9 @@
+package io.appform.dropwizard.actors.actor.metadata;
+
+import io.appform.dropwizard.actors.actor.MessageMetadata;
+
+public interface MessageMetadataGenerator {
+
+    void generate(final MessageMetaContext messageMetaContext, MessageMetadata messageMetadata);
```

Comment:
Why is one variable final, the other not? Also this looks like something that can have the @FunctionalInterface annotation